tests/dataframe/test_describe_pytest.py

```diff
@@ -28,7 +28,10 @@ def test_flights_describe(self):
         ed_flights = self.ed_flights()
 
         pd_describe = pd_flights.describe()
-        ed_describe = ed_flights.describe()
+        # We remove bool columns to match pandas output
```

Comment:
Can we verify the `bool` columns before we drop them to ensure things don't break down the line?

Reply:
I think those are covered in other `agg` tests. I am thinking it might be duplication